frontend/src/metabase/static-viz/lib/format.ts

```diff
@@ -30,6 +31,17 @@ import {
 import { rangeForValue } from "metabase-lib/queries/utils/range-for-value";
 import { getColumnKey } from "metabase-lib/queries/utils/get-column-key";
 
+const getRemappedValue = (
```

Comment:
This feels a bit out of place. I haven't tested this extensively. But it feels like we should be able to keep format code the same. The part where it remapped column value should happen before formatting.

I'm not sure why the app code doesn't need to change its format code while the static viz does.

Do you think it's possible?

Reply:
@WiNloSt it is not needed for the main app because a similar code is **inside** formatValue function